Advanced-Recruit-Training/src/LogFileReader.cpp

```diff
@@ -62,10 +62,28 @@ bool LogFileReader::parseLine(const QString& line, BatteryData& batteryData) con
     QString timeString = sections.at(0);
     batteryData.time = QTime::fromString(timeString, STRING_TIME_FORMAT);
 
-    batteryData.voltage = sections.at(1).toDouble();
+    if(sections.length() < COLUMNS || sections.length() > COLUMNS)
```

Comment:
You're essentially saying if the length of the section isn't COLUMNS return false so you can write this in a shorter if statement

Reply:
This is basically saying if sections.length() does not equal columns!